docs/src/components/ComponentDoc/ComponentDocAccessibility.tsx

```diff
@@ -45,29 +125,58 @@ export const ComponentDocAccessibility = ({ info }) => {
         </Text>
       )}
 
+      {((behaviorName && info.behaviors) ||
+        (behaviorName && accIssues) ||
+        (info.behaviors && accIssues)) && (
+        <ul>
+          <li>
+            Behaviors
+            <ul>
+              {behaviorName && <li>{link(`Default: ${behaviorName}`, '#default-behavior')} </li>}
+              {info.behaviors &&
+                getAvailableBehaviorsFromJson(info.behaviors).map(behavior => {
+                  return (
+                    <li>
+                      {link(`${behavior.name.replace('.ts', '')}`, `#${getId(behavior.name)}`)}
```

Comment:
These links do not point to the right location. E.g. we get `https://stardust-react-q53q0jbms.now.sh/#default-behavior`, but it should be `https://stardust-react-q53q0jbms.now.sh/components/alert/accessibility#default-behavior`

Reply:
Generic issue, bug reported:
https://github.com/stardust-ui/react/issues/1851